cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/util/OAuthUtils.java

```diff
@@ -31,11 +32,10 @@ private OAuthUtils() {}
      *
      * @param response http response
      * @param error error message
-     * @param status status code
      * @return a null view
      */
-    public static ModelAndView writeTextError(final HttpServletResponse response, final String error, final int status) {
-        return OAuthUtils.writeText(response, "error=" + error, status);
+    public static ModelAndView writeTextError(final HttpServletResponse response, final String error) {
+        return OAuthUtils.writeText(response, "error=" + error, HttpStatus.SC_BAD_REQUEST);
```

Comment:
For now, we return the access token as a plain/text. We'll do the same for the error page. But we'll need to provide a JSON capaibility in both cases.